### DISCUSSION TITLE
components.json base config for monorepos

### DISCUSSION THREAD

**Discussion by keeandev** [shadcn-ui/ui#656] [Ideas]
I've found myself trying to migrate to the new cli within my company's monorepo. I believe that the components.json should be able to extend existing configs so that they can be reused, possibly within a monorepo config package. Plenty of libraries resort to this solution when dealing with configurations. 

Thanks for reading! Would love to start a discussion about this concept.

Thanks,
DinoZoidDev

**Comment by ismoiliy98** [comment]
Maybe you can use [this](https://youtu.be/QJlTWj30krw) approach to be able to move your config as tailwind preset and plugin, so then you can extend/reuse tailwind config/plugin/preset in your apps inside your monorepo
